### ISSUE TITLE
Prepare qt6 version

### ISSUE THREAD

**Issue by IllDepence** [IllDepence/anki_add_pitch_plugin#36]
When running with `DISABLE_QT5_COMPAT=1` (tested on `Version ⁨2.1.55 (01caec2a)⁩` `Python 3.10.9` `Qt 6.4.1` `PyQt 6.4.0`)

the following deprecation warnings are shown

```  
util.py:36:'Qt.WindowModal' will stop working. Please use 'Qt.WindowModality.WindowModal' instead.
util.py:46:'QDialogButtonBox.Ok' will stop working. Please use 'QDialogButtonBox.StandardButton.Ok' instead.
util.py:46:'QDialogButtonBox.Cancel' will stop working. Please use 'QDialogButtonBox.StandardButton.Cancel' instead.
util.py:52:'QtClassProxy.exec_()' is deprecated. Please use 'QtClassProxy.exec()'
util.py:36:'Qt.WindowModal' will stop working. Please use 'Qt.WindowModality.WindowModal' instead.
util.py:46:'QDialogButtonBox.Ok' will stop working. Please use 'QDialogButtonBox.StandardButton.Ok' instead.
util.py:46:'QDialogButtonBox.Cancel' will stop working. Please use 'QDialogButtonBox.StandardButton.Cancel' instead.
util.py:52:'QtClassProxy.exec_()' is deprecated. Please use 'QtClassProxy.exec()'
```

**Comment by Weathercold** [comment]
I'm using Qt6 version with ``DISABLE_QT5_COMPAT=1`` now and the above warnings have become errors. Is there any plan to make the plugin compatible with Qt6?

**Comment by IllDepence** [comment]
Thanks for making me aware of that.

Should be fixed with 1437209. I made an updated version available on AnkiWeb for versions 2.1.62 upwards. Additionally [here as a GitHub release](https://github.com/IllDepence/anki_add_pitch_plugin/releases/tag/0.9.1) for an option to install from file.

Please let me know if it resolves the issue on your end.

**Comment by Weathercold** [comment]
Thanks! I can confirm that the crashes are resolved.
